packages/hosted-assets/README.md

```diff
@@ -22,22 +22,22 @@ asset-url("some/blob.png") // -> "https://<origin>/some/blob.png"
 ## If you want to use an image
 
 1. Make sure the image exists in the [`kaizen-design-system-assets` repo](https://github.com/cultureamp/kaizen-design-system-assets). If it doesn't, make a PR to add it (ask the Design Systems team for review). Images in this repo will be automatically uploaded and made available via a CloudFront distribution.
-2. Make sure that the `@cultureamp/kaizen-binary-assets` package is in your repo's `package.json`. This package provides you the `assetUrl` function that translates the path to the file in the `kaizen-design-system-assets` repo into its equivalent CloudFront URL.
-3. Import the `index.ts` or `index.scss` file and you can access the image through the `assetUrl`/`asset-url` function:
+2. Make sure that the `@kaizen/hosted-assets` package is in your repo's `package.json`. This package provides you the `assetUrl` function that translates the path to the file in the `kaizen-design-system-assets` repo into its equivalent CloudFront URL.
+3. Import the `@kaizen/hosted-assets` module and you can access the image through the `assetUrl`/`asset-url` function:
 
 ```js
-// JS
-import assetUrl from "@cultureamp/kaizen-binary-assets/lib/index.ts";
+// TS/JS
+import { assetUrl } from "@kaizen/hosted-assets";
 
-<SomeComponent image={assetUrl("illustrations/ham.png")} />
+<SomeComponent image={assetUrl("illustrations/tofu.png")} />
```

Comment:
Rest of the PR looks good except for the de-hamming, which I cannot support.

Reply:
@sebpearce I hope that this difficult compromise will ease the tensions between our people.

```tsx
<SomeComponent
  image={assetUrl("illustrations/plant-based-ham-substitute.png")}
/>
```

Reply:
I accept this generous peace offering and look forward to our mutually beneficial future.